app/src/main/res/layout-land/activity_signature.xml

```diff
@@ -98,6 +99,6 @@
 
     </androidx.constraintlayout.widget.ConstraintLayout>
 
-  </androidx.cardview.widget.CardView>
```

Comment:
We should consider adding lint checks to stop people from using the `androidx.cardview.widget.CardView` widget.

Reply:
Yep, we need to actually do that for few of the widgets that have MDC counter parts

Reply:
Cool, can we add a task in Clubhouse for this?